src/data/resources.js

```diff
@@ -551,4 +551,6 @@ const resources = [
     },
 ]
 
-export default resources;
\ No newline at end of file
+const resourceRequestLink = "https://docs.google.com/forms/d/1SFE8zabjOE662GENVG6L7TZ6qg-MQeqQElsxjaDzyMg/edit"
```

Comment:
This currently leads to the page allowing users to edit the form (if they're in the ACM Google Drive). No worries, because I've definitely made this mistake a many times :)

Can we replace it with this link? https://docs.google.com/forms/d/e/1FAIpQLSetpg4nnKfdhMRY_B_YytXl0fJSb8EEzGecO6ks0hx2SKb8OQ/viewform?usp=sf_link.